controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/TenantSerializer.java

```diff
@@ -215,7 +217,10 @@ private CloudTenant cloudTenantFrom(Inspector tenantObject) {
         Instant tenantRolesLastMaintained = SlimeUtils.instant(tenantObject.field(tenantRolesLastMaintainedField));
         List<CloudAccountInfo> cloudAccountInfos = cloudAccountsFromSlime(tenantObject.field(cloudAccountsField));
         Optional<BillingReference> billingReference = billingReferenceFrom(tenantObject.field(billingReferenceField));
-        return new CloudTenant(name, createdAt, lastLoginInfo, creator, developerKeys, info, tenantSecretStores, archiveAccess, invalidateUserSessionsBefore, tenantRolesLastMaintained, cloudAccountInfos, billingReference);
+        Optional<String> planId = planId(tenantObject.field(planIdField));
```

Comment:
Consider simplifying with something like
```suggestion
        Optional<String> planId = SlimeUtils.optionalString(tenantObject.field(planIdField));
```